sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventParticipantEditForm.java

```diff
@@ -103,6 +104,13 @@ protected void addFields() {
 
 		region.addItems(FacadeProvider.getRegionFacade().getAllActiveAsReference());
 
+		CountryReferenceDto countryDto = event.getEventLocation().getCountry();
```

Comment:
I think the proper logic here is to check whether region and district have been filled in for the event - that removes the need to check for the server country etc.

Reply:
My understanding, from the story, is that those fields should be required based on the server country, so what should I do? :)

Reply:
They need to be required when no region and district have been filled in for the event. While this is the case when a different country has been selected for the event, checking whether the region/district have been filled in is a) easier and b) less prone to errors than checking for the country again here.